packages/common/container-definitions/src/loader.ts

```diff
@@ -389,21 +389,48 @@ export interface IContainer extends IEventProvider<IContainerEvents> {
 	proposeCodeDetails(codeDetails: IFluidCodeDetails): Promise<boolean>;
 
 	/**
-	 * Attaches the Container to the Container specified by the given Request.
+	 * When a container is created in Fluid it starts in what we call a detached state,
+	 * which means it is detached from server storage and unavailable for other users to open or collaborate with.
+	 * A detached container can be serialized, stored, and rehydrated by the application.
 	 *
-	 * @privateRemarks
+	 * To finish the creation of a container and make it available for other users to open and collaborate with it must be attached,
+	 * which means it is attached to or persisted in server storage. This function is what accomplishes that.
+	 *
+	 * In order to craft the IRequest parameter necessary to call this function you should use functionality provided
+	 * by the driver you are using. This is usually a free function called createCreateNewRequest, which should take
+	 * strongly typed arguments related to your driver and server.
```

Comment:
Good paragraph, though it highlights how weird this flow is.  Rephrasing suggestion for active voice and avoiding hedging.
```suggestion
	 * Use the functionality provided by the driver to craft the IRequest parameter.
	 * This is usually a free function called createCreateNewRequest, which takes
	 * strongly typed arguments related to your driver and server.
```